README.md

```diff
@@ -22,9 +22,68 @@ Once you join the The [Presto Foundation Slack Workspace](https://join.slack.com
+| Rebecca Schlussel         |[rschlussel](https://.github.com/rschlussel)         | Facebook    | 1 Jan, 2020  | TBD          |
+| James Sun                 |[highker](https://github.com/highker)                | Facebook    | 1 Jan, 2020  | TBD          |
+| Wenlei Xie                |[wenleix](https://github.com/wenleix)                | Facebook    | 1 Jan, 2020  | TBD          |
+| Rongrong Zhong            |[rongrong](https://github.com/rongrong)              | Facebook    | 1 Jan, 2020  | TBD          |
+| Shixuan Fan               |[shixuan-fan](https://github.com/shixuan-fan)        | Facebook    | 1 Jan, 2020  | TBD          |
+| Jiexi Lin                 |[jessesleeping](https://github.com/jessesleeping)    | Facebook    | 1 Jan, 2020  | TBD          |
+
+Per the [Technical Charter](CHARTER.md), when there are less than six related companies represented on the TSC, the employees of each company may only submit one vote on TSC decisions where voting is required.
+
+## Policies and procedures
+
+The Presto TSC is governed by the [Presto Technical Charter](CHARTER.md).  The Charter provides foundational structure for the TSC on topics such as its scope, how to make decisions, and how to make changes to itself.  At the same time, it grants the TSC a high degree of freedom when determining how to implement the policies of the Presto Foundation.
+
+The following policies and procedures have been adopted by the TSC.
+
+### Term of TSC voting members
+
+The Presto TSC seeks to balance the energy and enthusiasm of new members with the stability provided by long-term members.  As such, TSC voting members will serve two-year terms, after which they will be eligible for re-election.  There is no limit to the number of times an individual may run for an open seat.
+
+To provide balance, the terms of TSC members will be adjusted so that roughly 50% of seats are elected each year.  If the classes become unbalanced, then the TSC shall designate the appropriate number of seats as one-year terms at the first opportunity to fix the imbalance.
+
+For simplicity, membership terms will be aligned to the calendar year, and will run from January 1st to December 31st of the following year. Initially, the TSC shall randomly designate 50% of the seats to be one-year terms, and the remainder shall be two-year terms.
+
+### Becoming a TSC Member
+
+The Technical Charter specifies that TSC voting members are initially the Presto committers.  Prior to each election, the TSC will determine the number of available seats and solicit nominations from the community.  The TSC has primary responsibility for ensuring there are sufficient candidates to fill each available seat.
+
+#### Eligibility
+
+Beginning with the 2021 election, any member of the technical community may stand for election to the TSC.  Membership in the Presto Foundation or status as a Presto committer are not pre-requisites.
```

Comment:
still think committer is a pre-requisite for TSC member

Reply:
May I suggest we add this to the agenda for the next TSC call (unless folks would like to discuss here as comments)?  I believe there are a range of opinions on this particular topic, and it would be good to reach an agreement.

Reply:
sure. go ahead